service/matching/poller/history.go

```diff
@@ -35,15 +35,15 @@ const (
 )
 
 type (
-	pollerIdentity string
+	Identity string
 
-	pollerInfo struct {
-		ratePerSecond  *float64
-		isolationGroup string
+	Info struct {
+		RatePerSecond  float64
```

Comment:
Might suggest we don't change this as part of a refactor unless we have to? It'll be tricky to track down if it causes problems.

Reply:
We have to change it, because otherwise we need to duplicate the definition of defaultTaskDispatchRPS in both tasklist package and poller package or export that constant, which is not ideal.